src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1033,7 +1033,7 @@ MulticopterAttitudeControl::control_attitude_rates(float dt)
 		_lp_filters_d[2].apply(rates(2)));
 
 	_att_control = rates_p_scaled.emult(rates_err) +
-		       _rates_int -
+               _rates_int +
```

Comment:
No! The derivative is computed using `rates_filtered - _rates_prev_filtered` which is the negative derivative. If you want to change the sign here, you would also have to compute the derivative using `_rates_prev_filtered - rates_filtered`.

Reply:
Thanks for clarification! I don't want to change sign here so this pr can be closed then.

Reply:
Just realised that my explanation was wrong (even if the result is the same). The correct one is: we compute the _positive_ derivative, but since it's made on the measurements and not on the error, you have to sum it with a negative sign. It's actually a "PI+D" and not a "PID" controller.